### DISCUSSION TITLE
Step by Step process to upgrade Snipe IT Version from v6.0.14 build 9038 (g163b3f6c0) to Latest stable available 6.2.4

### DISCUSSION THREAD

**Discussion by ravipawar23** [snipe/snipe-it#14073] [Q&A]
Dear Team,

can somebody help me to with Step-by-Step process to upgrade Snipe IT Version from v6.0.14 build 9038 (g163b3f6c0) to Latest stable available 6.2.4 on below link? 

https://github.com/snipe/snipe-it/releases/tag/v6.2.3

what all kind of prerequisites need to fulfill

i have tried with taking the VM snapshots but no luck it is giving me error of 404.8-page can't found.

my environment details are given in attachments.

VM - Windows Server 2022 Stand. 

! [2023-12-25 15_47_24-Admin for GitHub] (https://github.com/snipe/snipe-it/assets/124572447/9a8155a0-a870-4504-8855-eaea64bc03e0)

Many thanks in advance

**Comment by marcusmoore** [comment]
Hi @ravipawar23, I'm going to close this so we can continue the conversation in #13712
